exports/exports.go

```diff
@@ -42,11 +42,22 @@ func (e *Export) ExportRouter(r chi.Router) {
 	})
 }
 
+func SwitchUser(user middleware.User) models.User {
```

Comment:
What do you think about renaming this method to something like `mapUserToModelsUser()`.  I know its long-winded, but I think it communicates its purpose a little better.

Also, capitalizing the first letter exports the method outside of the package.  I do not think we need to export this method outside of the package at the moment.